dips/dip-4.md

```diff
@@ -154,6 +154,8 @@ RawTransaction {
 
 ## Refunds
 
+Refunds are now expected to follow [this document](https://github.com/diem/dip/tree/master/dips/refunds-addendum.md).
+
 Merchant B now wishes to refund user A. But user A was sending from a custodial account so merchant B must send the funds back to the custodial account and include subaddress information so that the funds are directed back to user A.  Merchant B’s client now constructs a transaction via the following where referenced_event is the committed event sequence number under the sending account of the original sent payment event:
```

Comment:
can you mark this as not recommended or deprecated? that's more clear in case people miss the link